jboss/container/eap/launch/ee/module.yaml

```diff
@@ -6,6 +6,22 @@ execute:
 - script: configure.sh
   user: '185'
 
+envs:
+    - name: LD_PRELOAD
```

Comment:
These env seem a duplicate of the one in https://github.com/jboss-container-images/jboss-eap-modules/blob/master/jboss/container/eap/launch/common/module.yaml#L10 
And this module depends on jboss.container.eap.launch.common so the env  can be removed.